README.md

```diff
@@ -399,7 +429,7 @@ $ curl -v --cacert certs/CA.crt  \
           https://MBP2021.local:1443/api/v1/ccs
 ```
 
-#### Response
+### Response
```

Comment:
This should stay at the `####` level since it only describes the JSON response, and restore the CBOR response above, so that they are both documented.